### PR TITLE
added the tan() function

### PR DESCRIPTION
**Pull request by Yash-Var**
### Type of Change

- Adding a new entry
- Editing an existing entry (fixing a typo, bug, issues, etc)
- Updating the documentation

### Checklist

- [ ] All writings are my own.
- [ ] My entry follows the Codecademy Docs style guide.
- [ ] My changes generate no new warnings.
- [ ] I have performed a self-review of my own writing and code.
- [ ] I have checked my entry and corrected any misspellings.
- [ ] I have made corresponding changes to the documentation if needed.
- [ ] I have confirmed my changes are not being pushed from my forked `main` branch.
- [ ] I have confirmed that I'm pushing from a new branch named after the changes I'm making.
- [ ] Under "Development" on the right, I have linked any issues that are relevant to this PR (write "Closes #<issue number> in the "Description" above).

### CODE REVIEW

**Comment by Yash-Var** [comment]
@SSwiniarski review it..

**Review by bcsamrudh — CHANGES_REQUESTED** [review]
@Yash-Var Codebyte should not contain the Output. You could always add a normal example along with the output.
 Also add a space after the ''' codebyte/php line, so that lint and format checks don't fail.

**Review by SSwiniarski — CHANGES_REQUESTED** [review]
@Yash-Var I did a first review and left a comment. Otherwise it looks good.

**Review by SSwiniarski — CHANGES_REQUESTED** [review]
@Yash-Var you still need to address my comment.

**Comment by Yash-Var** [comment]
@SSwiniarski  done

File: content/php/concepts/math-functions/terms/tan/tan.md [hunk 1]
```diff
@@ -0,0 +1,42 @@
...
+$tangent = tan($angle);
+
+echo "Tangent of $angle radians: $tangent\n";
+?>
+
+```
+## Output
+```Output/php
+Tangent of 0.8 radians: 1.0296385570504
+```
```
Comment (yangc95) [inline content/php/concepts/math-functions/terms/tan/tan.md:42]:
```suggestion

```

File: content/php/concepts/math-functions/terms/tan/tan.md [hunk 2]
```diff
@@ -0,0 +1,42 @@
...
+Using `tan()` to return the tangent of the angle of `0.8`:
+
+```codebyte/php
+<?php
+$angle = 0.8; // Angle in radians
+$tangent = tan($angle);
+
+echo "Tangent of $angle radians: $tangent\n";
+?>
+
```
Comment (yangc95) [inline content/php/concepts/math-functions/terms/tan/tan.md:37]:
```suggestion
```

File: content/php/concepts/math-functions/terms/tan/tan.md [hunk 3]
```diff
@@ -0,0 +1,42 @@
...
+## Codebyte Example
+
+Using `tan()` to return the tangent of the angle of `0.8`:
+
+```codebyte/php
+<?php
+$angle = 0.8; // Angle in radians
+$tangent = tan($angle);
+
+echo "Tangent of $angle radians: $tangent\n";
```
Comment (yangc95) [inline content/php/concepts/math-functions/terms/tan/tan.md:35]:
```suggestion
  $angle = 0.8; // Angle in radians
  $tangent = tan($angle);

  echo "Tangent of $angle radians: $tangent\n";
```

File: content/php/concepts/math-functions/terms/tan/tan.md [hunk 4]
```diff
@@ -0,0 +1,42 @@
...
+
+```pseudo
+$result = tan($angle);
+```
+
+Where `$angle` is the angle in radians.
+
+## Codebyte Example
+
+Using `tan()` to return the tangent of the angle of `0.8`:
```
Comment (yangc95) [inline content/php/concepts/math-functions/terms/tan/tan.md:28]:
```suggestion
Using `tan()` to return the tangent of an angle in radians:
```

File: content/php/concepts/math-functions/terms/tan/tan.md [hunk 5]
```diff
@@ -0,0 +1,42 @@
...
+  - 'Web Development'
+Tags:
+  - 'Arithmetic'
+  - 'Functions'
+CatalogContent:
+  - 'learn-php'
+  - 'paths/computer-science'
+---
+
+The **`tan()`** function  returns the tangent of the angle passed as the argument. The return value is a floating-point number.
```
Comment (yangc95) [inline content/php/concepts/math-functions/terms/tan/tan.md:16]:
```suggestion
The **`tan()`** function returns the tangent of the angle passed as the argument. The return value is a floating-point number.
```

**Review by yangc95 — COMMENTED** [review]
@Yash-Var Almost good to merge! Left a few single-suggestions

**Comment by Yash-Var** [comment]
@SSwiniarski @yangc95 done.

File: content/php/concepts/math-functions/terms/tan/tan.md [hunk 6]
```diff
@@ -0,0 +1,42 @@
...
+```codebyte/php
+<?php
+$angle = 0.8; // Angle in radians
+$tangent = tan($angle);
+
+echo "Tangent of $angle radians: $tangent\n";
+?>
+
+```
+## Output
```
Comment (SSwiniarski) [inline content/php/concepts/math-functions/terms/tan/tan.md:39]: You keep marking this resolved. Are you misunderstanding me? Codebytes should not be followed by an Output section. I want you to delete the Output header and the code block that follows it. I.e. I want you to remove the text entirely from the PR.

**Review by SSwiniarski — CHANGES_REQUESTED** [review]
@Yash-Var I checked again, and you keep marking my comment as resolved without doing what I am asking you to. Are you in need of some assistance?

File: content/php/concepts/math-functions/terms/tan/tan.md (hunk 6 above)
Comment (Yash-Var) [reply content/php/concepts/math-functions/terms/tan/tan.md]: @SSwiniarski now check i have solved

**Review by yangc95 — APPROVED** [review]

**Comment by yangc95** [comment]
@Yash-Var I approved of this PR after making a few minor direct changes and resolved any failing tests: https://github.com/Codecademy/docs/pull/2187/commits/12aee10c5de62c0fdddab32dbdeb895730e813ac, https://github.com/Codecademy/docs/pull/2187/commits/96dabfe31dfd2e2a0cbbcdc5a537f4460a3bbd92

I also noticed that you're using your main branch to create this PR. For future entries, make sure to create a **separate branch** before creating a new folder/file.

**Review by SSwiniarski — APPROVED** [review]
@Yash-Var looks good to merge.

**Comment by SSwiniarski** [comment]
@Yash-Var congrats on the new entry! Live link here: https://www.codecademy.com/resources/docs/php/math-functions/tan

image

**Pull request merged**